core/services/ocr2/plugins/ocr2keeper/evm21/registry.go

```diff
@@ -122,14 +127,26 @@ var upkeepStateEvents = []common.Hash{
 }
 
 type MercuryConfig struct {
-	cred *models.MercuryCredentials
-	abi  abi.ABI
+	Cred *models.MercuryCredentials
```

Comment:
If there is an accessor function, does this value need to be exported on the struct?

Reply:
that is right, the `Credentials()` is more than enough, will update.